crud/common/schema.lua

```diff
@@ -135,6 +135,25 @@ local function get_space_schema_hash(space)
     return digest.murmur(msgpack.encode(space_info))
 end
 
+function schema.filter_obj_fields(obj, field_names)
+    if field_names == nil or obj == nil then
+        return obj
+    end
+
+    local result = {}
+
+    for _, field_name in ipairs(field_names) do
+        result[field_name] = obj[field_name]
+        if result[field_name] == nil then
+            return nil, FilterFieldsError:new(
+                    'Object doesn\'t contain field named %q', field_name
+            )
+        end
```

Comment:
Will it work for nullable fields?

See similar issue - https://github.com/tarantool/crud/issues/140